multiuser/api/che-multiuser-api-resource/src/main/java/org/eclipse/che/multiuser/resource/api/usage/tracker/RamResourceUsageTracker.java

```diff
@@ -81,9 +86,18 @@ public RamResourceUsageTracker(
           if (startingEnvironment != null) {
             currentlyUsedRamMB += environmentRamCalculator.calculate(startingEnvironment);
           }
+          if (SidecarToolingWorkspaceUtil.isSidecarBasedWorkspace(config.getAttributes())) {
+            LOG.warn(
+                "Sidecar based workspace `{}` is starting, memory of its plugins is not taken in "
+                    + "account while workspace is starting. Set memory resources limits may work incorrectly");
+          }
+        } else {
+          // Estimation of memory for starting workspace with Devfile is not implemented yet
+          // just ignore such
+          LOG.warn(
+              "Devfile based workspace `{}` is starting, its memory is not taken in account "
```

Comment:
How about: ``"Memory consumption is not taken into account while a devfile-based workspace `{}` is starting. The memory limits may not be applied correctly."``